src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
Internal then? Because having a settable property worries me a bit.

(and it wouldn't be very consistent: https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.Cookies/Events/CookieSignedInContext.cs#L49)